ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkWriter.java

```diff
@@ -861,13 +877,13 @@ private void writeThreeWindingsTransformers(AmplExportContext context, TableForm
                         .writeCell(middleVlNum)
                         .writeCell(r2)
                         .writeCell(x2)
+                        .writeCell(g2)
                         .writeCell(0.0)
-                        .writeCell(0.0)
-                        .writeCell(0.0)
+                        .writeCell(b2)
                         .writeCell(0.0)
```

Comment:
Are you sure that the g2/b2 (resp. g3/b3) are exported at the good side. Looking to the column names, we have: `r, x, g1, g2, b1, b2`.
It seems that the side1 is always a read end, and the side2 is the middleBus, so I agree with your implementation, but I'm not totally confident and want to be sure it's OK

Reply:
I think that 1 and 2 are respectively the left side of the serie impedance and the right side of the serie impedance. So, I put everything on the left side (=network side). It is okay for you ?

Reply:
On tuesday, we discuss about it with @MioRtia: if the [scheme of the website](https://www.powsybl.org/pages/documentation/grid/model/index.html#three-windings-transformer), I totally agree with your proposal